client/css/style.css

```diff
@@ -144,7 +144,7 @@ button {
 
 code,
 pre,
-#chat .msg[data-type="motd"] .text,
+#chat .msg[data-type="monospace_block"] .text,
```

Comment:
Does anything need to be changed in morning too? https://github.com/thelounge/thelounge/blob/master/client/themes/morning.css#L127